app/scripts/views/sign_in_recovery_code.js

```diff
@@ -27,8 +28,10 @@ const View = FormView.extend({
 
     return account.consumeRecoveryCode(code)
       .then((result) => {
-        if (result.remaining < 1) {
-          // TODO Lets handle automatically generating recovery codes separately
+        if (result.remaining < 2) {
+          return this.navigate('/settings/two_step_authentication/recovery_codes', {
```

Comment:
I notice we use this string in two places. Worth defining in `lib/constants.js`? (I realise the other strings above aren't defined that way, but might be worth starting to move in that direction?)

Reply:
I think we'd be better off moving those not to constants.js, but to the router or a file that is just a list of routes. What I'd like to do is avoid the duplication of route names between the router, the views, and [get-frontend](https://github.com/hritvi/fxa-content-server/blob/c629fafc4ee0b345359c2e22a3bdfcbafc9ad774/server/lib/routes/get-frontend.js#L8) which enumerates the list of frontend routes for the server.

The idea is to create one central list of routes->path for the view. That's the longer term direction I'd like to go [over in the load views on demand PR](https://github.com/mozilla/fxa-content-server/pull/6480).

Reply:
BTW, trying to implement my idea is not the responsibility of this PR. Go forth with what's best here and I'll make my changes when I start the work I mention above!